config_files/public/local_config.ini

```diff
@@ -64,6 +66,29 @@ reserve_token_address         =
 reserve_token_name            = Kenyan Shilling
 reserve_token_symbol          = Ksh
 
+[CELO]
```

Comment:
Im getting
```
KeyError: 'CELO'
```
on the quick setup script because private key configs are missing.

Adding this to generate_secrets.py fixes it:
```

    CELO = 'CELO'
    add_val_sp(CELO, 'master_wallet_private_key', eth_pk())
    add_val_sp(CELO, 'owner_private_key', eth_pk())
    add_val_sp(CELO, 'float_private_key', eth_pk())
```

BUT, given that celo doesn't even work properly on a standard ganache chain, i think we should just remove it from the default configs entirely.

Reply:
Fair enough, nixed it from default configs. I only really added it there in the first place to serve as a demo of how multichain works